presto-docs/src/main/sphinx/security/tls.rst

```diff
@@ -27,7 +27,7 @@ that confirms the information is correct:
     Enter keystore password:
     Re-enter new password:
     What is your first and last name?
-      [Unknown]:  presto-coordinator
+      [Unknown]:  presto-coordinator-hostname.com
```

Comment:
(updating my previous comment)

Let's make this `presto-coordinator.example.com`, as the above is a potentially valid domain name that could belong to someone else. This is what we do in other documentation, such as `security/cli`, and is safe since `example.com` is reserved by [https://tools.ietf.org/html/rfc2606](RFC 2606) for this purpose.